src/steps/account/account-field-equals.ts

```diff
@@ -31,10 +31,11 @@ export class AccountFieldEquals extends BaseStep implements StepInterface {
     field: 'operator',
     type: FieldDefinition.Type.STRING,
     optionality: FieldDefinition.Optionality.OPTIONAL,
-    description: 'Check Logic (be, not be, contain, not contain, be greater than, or be less than)',
+    description: 'Check Logic (set to, not set to, containing, not containing, greater than, less than, be set, or not be set)',
   }, {
     field: 'expectedValue',
     type: FieldDefinition.Type.ANYSCALAR,
+    optionality: FieldDefinition.Optionality.OPTIONAL,
```

Comment:
One last comment that related to all steps:

- [ ] Because `expectedValue` is now optional, it would now be possible to run a step with `operator` set to `be` and `expectedValue` not set at all.  We should immediately respond with an error in those cases (e.g. when `expectedValue` is not set and `operator` isn't one of `set to)